cloudformation/prism-role.template

```diff
@@ -24,7 +24,8 @@
                       "ec2:Describe*",
                       "iam:Get*",
                       "iam:List*",
-                      "autoscaling:Describe*"
+                      "autoscaling:Describe*",
+                      "s3:List*"
```

Comment:
Looks like `"s3:ListAllMyBuckets"` and `"s3:GetBucketLocation"` are required here. `List*` gives too many permissions and won't be enough for this feature to work.